docker-compose.yml

```diff
@@ -0,0 +1,211 @@
+version: '3.8'
+
+services:
+  angular-build:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+    container_name: oppia-angular-build
+    volumes:
+      - .:/app/oppia
+      - shared_node_modules:/app/oppia/node_modules
+      - shared_third_party:/app/oppia/third_party
+      - shared_proto_files:/app/oppia/proto_files
```

Comment:
Why do we need proto files here?

Reply:
ng-build is compiling all .ts files, some ts files are importing the proto_files from /extension/classifiers/proto. so that's why need to to have this also, otherwise there is error while compiling ng-build as well as webpack

Reply:
`./proro_files` only contains Python files, right?
![image](https://github.com/oppia/oppia/assets/16653571/ba7462c9-010e-4714-89c0-697994479286)

Reply:
no, some proto files are also downloaded within /extenstions/classifiers/proto

Reply:
I agree we generate `*.ts` files in `/extenstions/classifiers/proto`, but my question is why do we need to mount `./proto_files` which only contains python files? I hope this helps understand the question better.

Reply:
no, updated the webpack and ng-build services